src/app/shared/document/document-detail/document-detail.component.ts

```diff
@@ -54,14 +56,17 @@ export class DocumentDetailComponent implements OnInit, OnDestroy {
   public createdBy$: Observable<string>;
   public updatedBy$: Observable<string>;
 
+  public readonly PERSPECTIVE_TABLE2 = Perspective.Table2;
```

Comment:
I would suggest referring to `Perspective.Table2` as table only across the application. The old one is there just not to break current smart document implementation.

Reply:
I would suggest to either remove the old table completely or rename it to TableBak or something. So we can get rid of the '2' character in general.

Reply:
Fixed